libs/accounts/two-factor/src/lib/backup-code.manager.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+interface BackupCodeRepository {
+  getBackupCodes(userId: string): string[];
+}
+
+export class BackupCodeManager {
```

Comment:
Are these intended to be consumed by NestJS? If so, we should add the @Injectable decorator